test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/ViewModels/ReadMePreviewViewModelTests.cs

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using NuGet.Common;
+using NuGet.PackageManagement.UI.ViewModels;
+using NuGet.Packaging;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.PackageManagement.UI.Test.ViewModels
+{
+    public class ReadMePreviewViewModelTests
+    {
+        [Fact]
+        public async void ReadMePreviewViewModelTests_NoPackagePathOrPackageId_NoErrorNoReadmeMarkdown()
```

Comment:
I'm not sure what others do, but I like the first part of the name to be whatever the "Act" does.
Also, missing Arrange/Act/Assert labels.

```suggestion
        public async void LoadReadme_NoPackagePathOrPackageId_NoErrorNoReadmeMarkdown()
```